web/core/src/jsMain/kotlin/androidx/compose/web/css/CSSProperties.kt

```diff
@@ -237,3 +237,70 @@ fun StyleBuilder.padding(value: CSSNumeric) {
     // padding hasn't Typed OM yet
     property("padding", value)
 }
+
+@Suppress("EqualsOrHashCode")
+data class CSSAnimation(
+    val keyframesName: String,
+    var duration: List<CSSSizeValue<out CSSUnitTime>>? = null,
+    var timingFunction: List<AnimationTimingFunction>? = null,
+    var delay: List<CSSSizeValue<out CSSUnitTime>>? = null,
+    var iterationCount: List<Int?>? = null,
+    var direction: List<AnimationDirection>? = null,
+    var fillMode: List<AnimationFillMode>? = null,
+    var playState: List<AnimationPlayState>? = null
+) : CSSStyleValue {
+    override fun toString(): String {
+        val values = listOfNotNull(
+            keyframesName,
+            duration?.joinToString(", "),
+            timingFunction?.joinToString(", "),
+            delay?.joinToString(", "),
+            iterationCount?.joinToString(", ") { it?.toString() ?: "infinite" },
+            direction?.joinToString(", "),
+            fillMode?.joinToString(", "),
+            playState?.joinToString(", ")
+        )
+        return values.joinToString(" ")
+    }
+}
+
+inline fun CSSAnimation.duration(vararg values: CSSSizeValue<out CSSUnitTime>) {
```

Comment:
It was intentionally because it isn't contract, just convenient way to modify structure. Ant it is very similar to how other properties added to CSS.

Reply:
You don't need chain here because it's intended to be called in 
```kotlin
animation(keyframe) {
   delay()
   duration()
}
```
So it's just a way how we are doing our DSL